src/js/rx/components/PrescriptionTable.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Link } from 'react-router';
+
+import SortableTable from '../../common/components/SortableTable';
+import { formatDate } from '../utils/helpers';
+import Prescription from './Prescription';
+
+class RefillStatus extends Prescription {
+  render() {
+    const { id } = this.props;
+
+    return (
+      <div key={id}>
```

Comment:
I don't think the key prop is supposed to be used here. Usually it's at the time you're creating a component as a part of a collection, so it would be a prop in the component created within the object in the map below.

https://facebook.github.io/react/docs/lists-and-keys.html#extracting-components-with-keys